code/style.css

```diff
@@ -1,22 +1,28 @@
 body {
```

Comment:
A little tip, instead of getting an undefined message if the user clicks on "Reveal the mystery" before they have been choosing a killer/weapon/room, an easy way to validate if all cards are choosen is for example by doing an if-else statement in this function!